packages/twenty-server/src/engine/core-modules/code-engine/utils/build-directory-manager.ts

```diff
@@ -0,0 +1,45 @@
+import { join } from 'path';
+import { tmpdir } from 'os';
+import fs from 'fs';
+
+import fsExtra from 'fs-extra';
+import { v4 } from 'uuid';
+
+const TEMPORARY_LAMBDA_FOLDER = 'twenty-build-lambda-temp-folder';
+const TEMPORARY_LAMBDA_SOURCE_FOLDER = 'src';
+const LAMBDA_ZIP_FILE_NAME = 'lambda.zip';
+const LAMBDA_ENTRY_FILE_NAME = 'index.js';
+
+export class BuildDirectoryManager {
```

Comment:
should not be in the utils, looks like a service to me! + lambda tied so it should be in a lambda section?

Reply:
ok